aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1496,6 +1500,7 @@ def __init__(
         serializer: Callable[[dict], str] | None = None,
         strip_prefixes: list[str | Pattern] | None = None,
         enable_validation: bool = False,
+        response_validation_error_http_status=None,
```

Comment:
```suggestion
        response_validation_error_http_code=None,
```

What do you think about this?

Reply:
Looks good